producers/mimosa26_sbg/src/M26Producer.cxx

```diff
@@ -0,0 +1,431 @@
+      if(!ret) executeError(cmd);
+
+
+      //m26_control->JTAG_Reset();
+      std::cout << " Send JTAG RESET to sensors " << std::endl;
+
+      reset_com();
+      cmd = "JTAGReset";
+
+      ret = IRC_RCBT2628__FRcSendCmdJtagReset ( CmdNumber/*not used*/, &DaqAnswer_CmdReceived, &DaqAnswer_CmdExecuted, 500 /* TimeOutMs */ );
+      if(ret) sendError(cmd);
+
+      ret = IRC_RCBT2628__FRcGetLastCmdError ( &VLastCmdError, 10000 /* TimeOutMs */ );
+      if(!ret) executeError(cmd);
+
+
+      eudaq::mSleep(200);
+
+      //m26_control->JTAG_Load(m_config);
+      std::cout << " JTAG sensors " << std::endl;
+      // JTAG sensors via mcf file
+
+      m_JTAG_file = m_config.Get("JTAG_file", "");
+      std::ifstream infile(m_JTAG_file.c_str());
+      if(!infile.good()) throw irc::InvalidConfig("Invalid path to JTAG file!");
+
+      /*debug*/ std::cout << "  -- JTAG file " << m_JTAG_file << std::endl;
+
+      reset_com();
+      cmd = "CmdJtagLoad";
+
+      ret = IRC_RCBT2628__FRcSendCmdJtagLoad ( CmdNumber/*not used*/, (char*)(m_JTAG_file.c_str()), &DaqAnswer_CmdReceived, &DaqAnswer_CmdExecuted, 100 /* TimeOutMs */ );
+      if(ret) sendError(cmd);
+
+      ret = IRC_RCBT2628__FRcGetLastCmdError ( &VLastCmdError, 10000 /* TimeOutMs */ );
+      if(!ret) executeError(cmd);
+
+
+      eudaq::mSleep(200);
+      //m26_control->JTAG_Start();
+      std::cout << " Send JTAG START to sensors " << std::endl;
+
+      reset_com();
+      cmd = "CmdJtagStart";
+
+      ret = IRC_RCBT2628__FRcSendCmdJtagStart ( CmdNumber/*not used*/, &DaqAnswer_CmdReceived, &DaqAnswer_CmdExecuted, 500 /* TimeOutMs */ );
+      if(ret) sendError(cmd);
+
+      ret = IRC_RCBT2628__FRcGetLastCmdError ( &VLastCmdError, 10000 /* TimeOutMs */ );
+      if(!ret) executeError(cmd);
+
+
+    } /* end if first time configured */
+
+    //m26_control->Configure_Run(m_config);
+    std::cout << " Configure Run " << std::endl;
+
+    reset_com();
+    cmd = "CmdRunConf";
+
+    IRC_RCBT2628__TCmdRunConf RunConf;
+
+    // FIXME get from m_config
+    RunConf.MapsName         = m_detectorTypeCode;
+    RunConf.MapsNb           = m_sumEnDetectors;
+    RunConf.RunNo            = 1; // FIXME If at all, must come from eudaq !! Finally, the controller does not need to know!!
+    RunConf.TotEvNb          = 0; // only used when saving to disk
+    RunConf.EvNbPerFile      = 0; // noy used by DAQ
+    RunConf.FrameNbPerAcq    = 1800; // 1800 is optimum for M26, 400 for M28
+    RunConf.DataTransferMode = 3; // 0 = No, 2 = EUDET2, 3 = EUDET3 // will steer which data is send to shrd mem. 2 ->all frames, 3-> only w/ trig
+    RunConf.TrigMode         = 0; // deprecated
+    RunConf.SaveToDisk       = 0; // 0 = No, 1 = Multithreading, 2 = Normal
+    RunConf.SendOnEth        = 0; // deprecated, not need with eudaq
+    RunConf.SendOnEthPCent   = 0; // deprecated, not need with eudaq
+
+    sprintf ( RunConf.DestDir, "c:\\Data\\test\\1" );

```

Comment:
Data is not yet returned from the actual IPHC DAQ module, so for now basically everything is more or less a hack.
Before we finally merge this when things are working this obviously needs some cleanup  - but I wouldn't worry about it in this stage.